samza-sql/src/main/java/org/apache/samza/sql/interfaces/SqlSystemSourceConfig.java

```diff
@@ -126,4 +131,12 @@ public Config getConfig() {
   public String getSource() {
     return source;
   }
+
+  public boolean isTable() {
```

Comment:
Do we need a separate isTable API here or can we use getTableDescriptor to return Optional.empty() if it is not a table?

Reply:
I can get rid of isTable